docs/integrations/microsoft-azure/azure-automation.md

```diff
@@ -43,7 +43,7 @@ In this section, you will configure a pipeline for shipping metrics from Azure M
 
 In this section, you will configure a pipeline for shipping diagnostic logs from Azure Monitor to an Event Hub.
 
-1. To set up the Azure Event Hubs cloud-to-cloud source in Sumo Logic portal, refer to our [Azure Event Hubs source documentation](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/).
+1. To set up the Azure Event Hubs source in Sumo Logic portal, refer to our [Azure Event Hubs documentation](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
```

Comment:
We should be specific since this source is only for logs
Azure Event Hubs documentation => Azure Event Hubs Source for Logs
Needs to be fixed in other pages as well